libs/agno/agno/os/router.py

```diff
@@ -1464,6 +1486,14 @@ async def create_workflow_run(
     ):
         kwargs = await _get_request_kwargs(request, create_workflow_run)
 
+        if hasattr(request.state, "user_id"):
+            user_id = request.state.user_id
+        if hasattr(request.state, "session_id"):
+            session_id = request.state.session_id
+        if hasattr(request.state, "dependencies"):
+            dependencies = request.state.dependencies
+            kwargs["dependencies"] = dependencies
```

Comment:
Are we sure we want to overwrite the query params like this? I'd expect the query params to take precedence

Reply:
I'd expect middleware to take presedence? Since it gets it from the token, which should be the highest authority. Regardless the user would be weird to use both?

Reply:
Done